themes/edx.org/lms/templates/header.html

```diff
@@ -44,18 +44,113 @@
 % endif
 </%block>
 
-% if uses_pattern_library:
+% if uses_bootstrap:
+<header class="navigation-container header-global ${"slim" if course and not disable_courseware_header else ""}" aria-label="Main" role="banner">
+% elif uses_pattern_library:
 <header class="header-global ${"slim" if course and not disable_courseware_header else ""}" aria-label="Main" role="banner">
 % else:
 <header class="${"header-global slim" if course and not disable_courseware_header else "header-global-new"}" aria-label="Main" role="banner">
 % endif
-  <div class="${"rwd" if responsive else ""} wrapper-header nav-container">
 
+% if uses_bootstrap:
+<nav class="navbar navbar-toggleable-md navbar-light" aria-label="${_('Main')}">
```

Comment:
It's tricky because so many elements have new classes (nearly every element) and an inline class="${"item-link" if uses_bootstrap else "item"}" will litter the code and be tricky to clean up later. I would agree with the sentiment that we deal with this when translating to a real page. I manually tested it and all looked good - although, agreed, a more detailed analysis can come at the next step.